js/components/ManuscriptsLoader.js

```diff
@@ -1,42 +1,231 @@
 import React from "react";
 
+/* ManuscriptsLoader - Renders the list of available manuscripts in the
+ * ManuscriptsMenu form component, and passes their selection/deselection
+ * events up the form component chain.
+ * It doesn't actually load the manuscripts list from an external source;
+ * this data has already been queried via the REST API by the App component.
+ * This does however contain the defaultManuscripts array, which the App
+ * component uses if the REST API is not available.
+ */
+
 /* Mock data to use in development when local API backend is not available */
-const fallbackManuscripts = [
-  { id: 20, shelfmark: "BL. Add. 12146", date: "1007 " },
-  { id: 21, shelfmark: "BL. Or. 8732", date: "770" },
-  { id: 22, shelfmark: "BL. Or. 8606", date: "723" },
-  { id: 23, shelfmark: "BL. Add. 17102", date: "598-599" },
-  { id: 24, shelfmark: "BL. Add. 17107", date: "540-541" },
-  { id: 25, shelfmark: "BL. Add. 12175", date: "533-534 " },
-  { id: 26, shelfmark: "BL. Add. 17109", date: "873 - 874" }
+export const defaultManuscripts = [
```

Comment:
For cleanliness, it would be nice for this to be in a separate json file that is imported into the component.

Reply:
I think I'll probably just move this to the tests folder and limit its use to coverage testing. I don't really see any benefit to displaying a canned list of manuscripts if the API is down, because the rest of the site won't work anyway if the API is down... This differs from the situation with the letters, where we've augmented the data we could get from the API with more info, and also the letter list is unlikely to change over the course of the project.